lnwire/update_add_htlc.go

```diff
@@ -72,6 +84,10 @@ type UpdateAddHTLC struct {
 	// next hop for this htlc.
 	BlindingPoint BlindingPointRecord
 
+	// CustomRecordsBlob is an optional blob of custom records that can be
+	// included in the ExtraData field of the UpdateAddHTLC message.
+	CustomRecordsBlob CustomRecordsBlob
```

Comment:
I think this should just be based off of `ExtraData`. As otherwise, right now it assumes that there's a single overarching TLV blob, but we'll just want to store everything in `ExtraData` below.